pkg/admin/transport/handler_user_import.go

```diff
@@ -0,0 +1,50 @@
+package transport
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/authgear/authgear-server/pkg/api"
+	"github.com/authgear/authgear-server/pkg/lib/userimport"
+	"github.com/authgear/authgear-server/pkg/util/httproute"
+	"github.com/authgear/authgear-server/pkg/util/httputil"
+)
+
+func ConfigureUserImportRoute(route httproute.Route) httproute.Route {
+	return route.WithMethods("POST").
+		WithPathPattern("/_api/admin/users/import")
+}
+
+type UserImportService interface {
+	ImportRecords(ctx context.Context, request *userimport.Request) (*userimport.Summary, []userimport.Detail)
+}
+
+type UserImportHandler struct {
+	JSON              JSONResponseWriter
+	UserImportService UserImportService
+}
+
+func (h *UserImportHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	var err error
+	var request userimport.Request
+	err = httputil.BindJSONBody(r, w, userimport.RequestSchema.Validator(), &request)
```

Comment:
Do we have a limit on the json size?

Reply:
Yes. pkg/util/httputil/json.go:16

Reply:
But it seems too large for user import. Maybe I refactor a bit to make it configurable.

Reply:
Fixed.